lib/libutils/ext/include/confine_array_index.h

```diff
@@ -142,4 +142,11 @@ static inline size_t confine_array_index(size_t index, size_t size) {
   return safe_index;
 }
 #endif
+#ifdef __riscv
+static inline size_t confine_array_index(size_t index, size_t size) {
+	(void)index;
+	(void)size;
```

Comment:
Risc-V is not immune to Spectre V1 type of attacks. Please try to provide a real implementation for this function.